### ISSUE TITLE
i3lock-fancy crashes if busybox is installed

### ISSUE THREAD

**Issue by tyrion** [NixOS/nixpkgs#70162]
**Describe the bug**
i3lock-fancy crashes if busybox is installed.

**To Reproduce**
```
$ i3lock-fancy
mktemp: unrecognized option '--suffix=.png'
BusyBox v1.30.1 () multi-call binary.

Usage: mktemp [-dt] [-p DIR] [TEMPLATE]

Create a temporary file with name based on TEMPLATE and print its name.
TEMPLATE must end with XXXXXX (e.g. [/dir/]nameXXXXXX).
Without TEMPLATE, -t tmp.XXXXXX is assumed.

        -d      Make directory, not file
        -q      Fail silently on errors
        -t      Prepend base directory name to TEMPLATE
        -p DIR  Use DIR as a base directory (implies -t)
        -u      Do not create anything; print a name

Base directory is: -p DIR, else $TMPDIR, else /tmp
```
Uninstalling busybox solves the problem.

**Expected behavior**
i3lock-fancy should not crash.

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.75, NixOS, 20.03pre194293.2436c27541b (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3`
 - channels(root): `"nixos-20.03pre194293.2436c27541b"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by worldofpeace** [comment]
I'm not sure why you need `busybox` installed on nixos, but the issue appears to be that `busybox` `mktemp` doesn't accept `--suffix`.
To prevent this we could hardcode 
* https://github.com/meskarune/i3lock-fancy/blob/1734f7752f1a323508584ccaef0a946f2feec1bd/i3lock-fancy#L10
to the specific `mktemp` nix store path.

**Comment by rnhmjoj** [comment]
Is `--suffix` coreutils specific?
If not I wouldn't hardcode coreutils in but lower the priority of busybox instead.

**Comment by TethysSvensson** [comment]
This was fixed in 55fb13e.
